articles/plural_markets_and_the_creative_uncertainty_of_open_source/open_source_equity.md

```diff
@@ -0,0 +1,253 @@
+
+![Limit order market book](marketbook.png)
+
+In another departure from the design of traditional company equity markets, the market book for equity tokens must make the identities of the members behind sell side limit orders public. This is a consequence of the facts that (a) shareholder voting power must be public and (b) shareholder voting power depends on the self-assessed value of equity tokens (which is expressed by the price in a sell side limit order). This provides transparency to the community as to who sponsors and influences any given open source project.
+
+I leave it as an open question whether the identities of buy side limit orders should be kept anonymous or not.
+
+### Disarming Sybil behavior
+
+Let us first recall why the standard QV rule, $v_{i,x}=\sqrt{s_{i,x}}$, is vulnerable to Sybil attacks. If an attacker with a fixed dollar budget $D$ distributes their capital uniformly across $n$ Sybils, they aggregate a total voting power of
+
+$$\sum_{i=1}^n\sqrt{s_{i,x}}
+     =\sum_{i=1}^n\sqrt{\frac{D}{n}}
+     =n\sqrt{\frac{D}{n}}=\sqrt{nD}$$
+
+This quantity scales with the number of Sybils, $n$, thus making Sybil behavior profitable even under a fixed dollar budget.
+
+In contrast, let us consider the voting rule
+$$v_{i,x}=s_{i,x}^\alpha \cdot h_i^\beta$$
+An attacker can distribute their budget $D$ across the equity ownership $s_{i,x}$ (for all $i$) and the community contributions $h_i$ (for all $i$) of the Sybils $i$ that they control, as long as the constraint on the budget $b$ is respected:
+$$b=\sum_i ps_{i,x}+\sum_i h_i \le D$$
+where $p$ is the average dollar price of the equity token in question.
+
+In doing so, the attacker seeks to maximize the total voting power of all Sybils:
+$$v = \sum_i s_{i,x}^\alpha \cdot h_i^\beta$$
+This is an optimization problem which can be solved using the method of [Lagrange multipliers](https://en.wikipedia.org/wiki/Lagrange_multiplier), whereby maximum voting power is attained when the partial derivatives of the Lagrangian $L=v-\lambda b$ vanish:
+$$\frac{\partial L}{\partial s_{i,x}}=\alpha s_{i,x}^{\alpha-1}h_i^\beta - \lambda p = 0$$
+$$\frac{\partial L}{\partial h_i}=\beta s_{i,x}^\alpha h_i^{\beta-1} - \lambda = 0$$
+$$\frac{\partial L}{\partial \lambda}= \sum_i ps_{i,x}+\sum_i h_i - D = 0$$
+
+The first two equations imply $h_i=p\frac{\alpha}{\beta}\cdot s_{i,x}$.
```

Comment:
I think this should be:

$h_i=p\frac{\beta}{\alpha}\cdot s_{i,x}$

($\alpha$ and $\beta$ are swapped)

Reply:
Yep. Fixed it. Thanks!